File: 29-01-2024/Questions/Python Solution/Solution2.py
l = []
i = 0
income = 0
flag = 1

while True:
    val = input()

    if val == '':
        break
    else:
        l.append(int(val))

    if l[i]<1 or l[i]>120:
        flag=0
        break

    elif l[i]<17:
        income+=200
    
    elif l[i]<=40:
        income+=400
    
    else:
        income+=300
    
    i=i+1
    
    if i==20:
        flag=0
        break

if flag==0:
    print('INVALID INPUT')

else:
    print('Total income '+str(income)+' INR')

File: 29-01-2024/Questions/Cpp Solution/Solution2.cpp
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int age, total = 0, i = 0;
    char str[10];
    while (1)
    {
        gets(str);
        if (str[0] == '\0')
        {
            break;
        }
        else
        {
            age = atoi(str);
        }
        if (age < 1 || age > 120)
        {
            printf("INVALID INPUT");
            return 0;
        }
        else if (age < 17)
        {
            total += 200;
        }
        else if (age <= 40)
        {
            total += 400;
        }
        else
        {
            total += 300;
        }
        i++;
        if (i == 20)
            break;
    }
    printf("Total Income %d INR", total);
}
